argo-cd-apps/base/member/infra-deployments/backup/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- backup.yaml
+components:
+  - ../../../../k-components/deploy-to-member-cluster-merge-generator
+  
```

Comment:
nit: you have an empty line but there are trailing spaces, just remove the trailing spaces. That is why GitHub display this red symbol here